client/python/tox.ini

```diff
@@ -3,6 +3,8 @@ isolated_build = true
 envlist =
     format
     py38
+    py39
```

Comment:
I'm not that familar with tox list but shouldn't we also add this to third_party/airflow/tox.ini also?  

It seems to work without it though..

Reply:
> The Python version to use for a given environment can be controlled via the base_python configuration, however if not set will try to use the environment name to determine something sensible: if the name is in the format of pyxy then tox will create an environment with CPython with version x.y (for example py310 means CPython 3.10). If the name does not match this pattern it will use a virtual environment with the same Python version as the one tox is installed into (this is the case for format).
- https://tox.wiki/en/latest/user_guide.html

It looks like because of the name tox knows what to do